resolwe/flow/managers/dispatcher.py

```diff
@@ -303,14 +303,18 @@ def _get_listener_settings(
         listener_settings = getattr(settings, "FLOW_EXECUTOR", {}).get(
             "LISTENER_CONNECTION", {}
         )
+        container_listener_connection = getattr(
+            settings,
+            "COMMUNICATION_CONTAINER_LISTENER_CONNECTION",
+            {"local": "127.0.0.1"},
```

Comment:
Try to avoid putting default values into the code. One option is to add a sanity check at the startup to verify that the setting is set, and the second option would be to add a dedicated file for default settings (e.g. `flow/setings.py`).